### PR TITLE
fix: 🐛 Fix SCSS lint issues

### PR DESCRIPTION
**Pull request by calcaide**
## Description

Fix SCSS lint issues

### Screenshots (if appropriate):

CI failing because the lint issues before the changes:
Screen Shot 2022-10-04 at 11 48 41 AM

### CODE REVIEW

File: ui/admin/app/styles/app.scss [hunk 1]
```diff
@@ -398,51 +398,53 @@
...
     border: sizing.rems(xxxxs) solid var(--ui-gray-subtler-2);
+    color: var(--failure);
   }
 
   .worker-auth-token {
-    display: flex;
-    gap: sizing.rems(m);
     align-items: flex-end;
+    display: flex;
+    gap: sizing.rems(m); // sass-lint:disable-line no-misspelled-properties
```
Comment (calcaide) [inline ui/admin/app/styles/app.scss:417]: @ZedLi linter says `gap` is misspelled, suggests to use `row-gap`. I have no context with the implementation of these styles, so I wanted to check with you before changing the `gap` to `row-gap` or to discuss possible solutions.

Comment (dvonanderson) [reply ui/admin/app/styles/app.scss:417]: I think that `gap` allows you to set both spacing for the `column`  and `row` without having to specifically say `column-gap` or `row-gap`

Comment (ZedLi) [reply ui/admin/app/styles/app.scss:417]: Yes that's right, it's just shorthand for both `row-gap` and `column-gap`, I wonder if our linter needs to just be updated to pick the syntax?

**Review by ZedLi — APPROVED** [review]
LGTM, thanks for fixing these warnings, totally missed them!

**Review by cameronperera — APPROVED** [review]
LGTM. I did a little digging and it looks like this is an issue for `row-gap` as well. Unfortunately, no activity on [this issue](https://github.com/sasstools/sass-lint/issues/1220).

**Comment by ZedLi** [comment]
> LGTM. I did a little digging and it looks like this is an issue for `row-gap` as well. Unfortunately, no activity on [this issue](https://github.com/sasstools/sass-lint/issues/1220).

Ahh this is an unmaintained repo since 2017...We should probably migrate off of this perhaps to something like `stylelint`?

**Comment by cameronperera** [comment]
> > LGTM. I did a little digging and it looks like this is an issue for `row-gap` as well. Unfortunately, no activity on [this issue](https://github.com/sasstools/sass-lint/issues/1220).
> 
> Ahh this is an unmaintained repo since 2017...We should probably migrate off of this perhaps to something like `stylelint`?

Potentially. I can try to migrate locally and see how that goes.

**Review by randallmorey — APPROVED** [review]

File: ui/admin/app/styles/app.scss (hunk 1 above)
Comment (calcaide) [reply ui/admin/app/styles/app.scss:417]: All good! 😄 In this case I just add the one liner to deactivate such rule. Not a fan of this practice but the code makes sense, so better to wait for the linter to be updated (they are already working on it).
Thanks for the info guys 👍

**Pull request merged**